### PR TITLE
Change status bar color to black

### PR DESCRIPTION
**Pull request by spiralhalo**
for better eye comfort

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorPrimary">#FF837D87</color>
```
Comment (nilesr) [inline app/src/main/res/values/colors.xml]: what the fuck is going on here

**Review by nilesr — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (nilesr) [reply app/src/main/res/values/colors.xml]: I mean with the 32-bit color instead of regular 24-bit color

Comment (spiralhalo) [reply app/src/main/res/values/colors.xml]: huh i didn't notice that. i just copied the topaz color from ColorPickerFragment. what do i do to make it right

Comment (nilesr) [reply app/src/main/res/values/colors.xml]: probably drop the leading FF which leaves [#837D87](https://color-hex.org/color/837d87)

**Comment by nilesr** [comment]
lookin good

**Comment by spiralhalo** [comment]
It works on mine so I'm about to look into it but seems like you had a better idea in place. Thanks Lain

**Comment by nilesr** [comment]
yeah pull down the latest changes

2017-12-03 21:27 GMT-08:00 spiralhalo <notifications@github.com>:

**Pull request merged**